src/components/ebay-filter-menu/README.md

```diff
@@ -0,0 +1,49 @@
+# ebay-filter-menu
+
+The `ebay-filter-menu` component is used as a checkbox menu specificially styled for filtering actions. Generally it is coupled with the `ebay-filter-menu-button` to create a selectable set of items by which to filter.
+
+## ebay-filter-menu Tag
+
+### ebay-filter-menu Usage
+
+```marko
+<ebay-filter-menu>
+    <ebay-filter-menu-item value="item 1">item 1</ebay-filter-menu-item>
+    <ebay-filter-menu-item value="item 2">item 2</ebay-filter-menu-item>
+    <ebay-filter-menu-item value="item 3">item 3</ebay-filter-menu-item>
+</ebay-filter-menu>
+```
+
+### ebay-filter-menu Attributes
```

Comment:
Do you want to update the documentation for `class-prefix` or just keep it as an internal attribute used by our components?

Reply:
I can add it. Most people won't need to use it, though.